model/task/task_test.go

```diff
@@ -3496,6 +3496,23 @@ func TestAddParentDisplayTasks(t *testing.T) {
 	assert.Equal(t, dt2.Id, tasks[3].DisplayTask.Id)
 }
 
+func TestSetCheckRunId(t *testing.T) {
+	require.NoError(t, db.Clear(Collection))
+	t1 := &Task{
+		Id: "t1",
+	}
+
+	assert.NoError(t, t1.Insert())
+	assert.NoError(t, t1.SetCheckRunId(12345))
+
+	var err error
+	t1, err = FindOneId(t1.Id)
```

Comment:
Have to `require.NotNil(t, t1)` or else this test can panic if it doesn't find the task.